cilium/README.md

```diff
@@ -43,6 +42,28 @@ To collect the cluster-wide `cilium-operator` metrics, see the [Cluster Agent][9
 
 2. [Restart the Agent][5].
 
+#### Log Collection
+Cilium contains two types of logs, `cilium-agent` logs and `cilium-operator` logs.
+
+#### Available for Agent >6.0
+1. Collecting logs is disabled by default in the Datadog Agent. Enable it in your [daemonset configuration][4]:
+
+    ```
+      (...)
+        env:
+          (...)
+          - name: DD_LOGS_ENABLED
+              value: "true"
+          - name: DD_LOGS_CONFIG_CONTAINER_COLLECT_ALL
+              value: "true"
+      (...)
+    ```
+
+2. Make sure that the Docker socket is mounted to the Datadog Agent as done in [this manifest][5] or mount the `/var/log/pods` directory if you are not using docker.
```

Comment:
```suggestion
2. Mount the Docker socket to the Datadog Agent as done in [this manifest][5] or mount the `/var/log/pods` directory if you are not using Docker.
```
The link points to `https://docs.datadoghq.com/agent/guide/agent-commands/#start-stop-and-restart-the-agent`